wpi.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+# This script performs WPI via dljc on a given project directory.
+# The inputs are similar to wpi-many.sh, which uses this script internally.
+# The only difference is that wpi-many.sh takes a list of projects, while
+# this script operates on a single project at a time.
+# See the documentation of wpi-many.sh for information on the inputs to this
+# script.
+#
+# Input differences compared to wpi-many.sh:
+# -i and -o are not valid options
+# new required option -d: the directory containing the target project
+#
+# The DLJC environment variable must point to the dljc script.
```

Comment:
Rather than having this requirement, this script (rather than `wpi-many.sh` could set the variable.  That simplifies use of the script.